site/content/docs/howto/supervisor/configure-supervisor-federationdomain-idps.md

```diff
@@ -0,0 +1,403 @@
+   not allow any users to authenticate, and a error message will be shown in its `status`.
+
+2. When `spec.identityProviders` is explicitly configured on a FederationDomain, then the FederationDomain will
+   allow clients to use any of those identity providers to authenticate. In this case, you may optionally also configure
+   identity transformations and policies that the FederationDomain should apply to each of these identity providers
+   (see below for details). When using the `pinniped get kubeconfig` CLI command, you will need to choose for
+   which identity provider you would like to generate a kubeconfig. A cluster may have multiple kubeconfigs,
+   e.g. one for each identity provider.
+
+The remainder of this guide focuses on the second case, and describes the settings that may be used to explicitly
+configure which identity providers are used, along with optional identity transformations and policies.
+
+## Configuring a FederationDomain's identity providers
+
+A user may authenticate to a FederationDomain using any of the IDPs configured in the FederationDomain's
+`spec.identityProviders`. To add IDPs to this list, simply configure each as a reference to the type and name
+of the resource. The identity provider resources must be in the same namespace where the Supervisor was installed.
+
+Here is an example FederationDomain with two IDPs configured.
+
+```yaml
+apiVersion: config.supervisor.pinniped.dev/v1alpha1
+kind: FederationDomain
+metadata:
+  name: my-provider
+  # Must be in the same namespace where the Supervisor is installed.
+  namespace: pinniped-supervisor
+spec:
+  issuer: https://my-issuer.example.com/any/path
+  tls:
+    secretName: my-tls-cert-secret
+  # Available identity providers are selected here...
+  identityProviders:
+    - displayName: ActiveDirectory for Admins
+      objectRef:
+        apiGroup: idp.supervisor.pinniped.dev
+        kind: ActiveDirectoryIdentityProvider
+        name: ad-for-admins
+    - displayName: Okta for Developers
+      objectRef:
+        apiGroup: idp.supervisor.pinniped.dev
+        kind: OIDCIdentityProvider
+        name: okta-for-developers
+```
+
+Now users may use either of the above identity providers to authenticate. You can create
+kubeconfigs for both IDPs for each cluster by using `pinniped get kubeconfig` twice for
+each cluster.
+
+## Important consideration when using multiple identity providers: conflicting usernames and group names
+
+When multiple identity providers are configured onto a FederationDomain, then a user may use any of those
+providers to authenticate to that FederationDomain. Since an identity in Kubernetes is simply a username string
+and a list of group name strings, it is very important to consider what will happen if two users each authenticate
+to a FederationDomain using different identity providers.
+
+1. If two users are assigned the same username string as a result of authenticating, then those two users will be
```

Comment:
This is a great section.  In the future I'm a bit inclined to pull it out into a separate page, with a link in the menu, perhaps under a heading `Security Features`.  I think if we can bubble up the interesting security features Pinniped provides and make them easy to find it will benefit the project.

I'd add things like `kubeconfig generation` to this sub-menu as well, discussing how credentials are not stored in the kubeconfig, how it can (usually) be shared safely as it contains minimal information, etc.

Reply:
Let's consider reorganizing this new page and the pre-existing "Required Configuration" page in a future PR. For this PR, I just want to get the information about how to use the new features onto the site.